packages/adblocker/test/utils.test.ts

```diff
@@ -262,4 +265,20 @@ describe('utils.ts', () => {
       expect(binSearch(array, 42)).to.equal(array.length - 1);
     });
   });
+
+  it('handles string utils', () => {
```

Comment:
Nit. Let's create an `it(...)` scope for each individual string utility.

Reply:
I think what Remi's meant it to have separate tests, eg.
`it('#findLastIndexOfUnescapedCharacter'`
and 
`it('#findIndexOfUnescapedCharacter`